report/src/main/kotlin/com/hartwig/actin/report/pdf/tables/trial/EligibleTrialGenerator.kt

```diff
@@ -134,17 +131,15 @@ class EligibleTrialGenerator(
             val title = "$trialDescriptionString that are open and potentially eligible $cohortsFromTrialsString"
 
             val footNote = listOfNotNull(
```

Comment:
This variable should be called `footNotes` I believe since it could contain multiple ?